producers.md

```diff
@@ -492,6 +502,17 @@ It includes a module to work with
 
 **[RSpec TAP](https://github.com/appleton/rspec_tap)** is a formatter for [RSpec](http://rspec.info/) which outputs TAP.
 
+## <a id="ruby"></a> [Rust](#rust)
+
+> Rust is a multi-paradigm system programming language focused on safety,
+> especially safe concurrency. Rust is syntactically similar to C++,
+> but is designed to provide better memory safety while maintaining high
+> performance.
+>
+> *according to [wikipedia](https://en.wikipedia.org/wiki/Rust_(programming_language))*
+
+[testanything](https://crates.io/crates/testanything) is a TAP producer for the [rust](http://rust-lang.org/) programming langage.
```

Comment:
Could you capitalize Rust in this sentence? I think that would fit better with their documentation style.

Reply:
ok